seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -93,10 +115,12 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
          |      receiveBuf = null;
          |    },
          |    send: function(msg) {
-         |      sendMsgBufIn.push(msg);
+         |      var encodedMsg =
+         |        msg.split("\7").join("\7\7").split("\0").join("\7");
```

Comment:
There also exists `replace` in JS. The above code would give `msg.replace("\7", "\7\7").replace("\0", "\7")`, and my suggested alternative would be `msg.replace("\\", "\\\\").replace("\0", "\\0")`